lib/fiware-iotagent-lib.js

```diff
@@ -177,20 +199,115 @@ function deactivate(callback) {
     contextServer.stop(callback);
 }
 
+function update(entityName, attributes, typeInformation, token, callback) {
+    ngsi.update(entityName, attributes, typeInformation, token, callback);
+}
+
+function query(entityName, attributes, typeInformation, token, callback) {
+    ngsi.query(entityName, attributes, typeInformation, token, callback);
+
```

Comment:
not needed line, please use a Linter

Reply:
Indeed, not needed code... why this? (and all the rest, seems pretty much out of scope of the PR)